client/src/pages/EditProfile/EditProfileForm/EditProfileForm.tsx

```diff
@@ -0,0 +1,96 @@
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Box from '@material-ui/core/Box';
+import { Formik, FormikHelpers } from 'formik';
+import * as Yup from 'yup';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import { CircularProgress } from '@material-ui/core';
+import { User } from '../../../interface/User';
+interface Props {
+  handleSubmit: (
+    {
+      newUsername,
+      newEmail,
+    }: {
+      newUsername: string;
+      newEmail: string;
+    },
```

Comment:
Refactoring this into a quick interface would improve the readability of this code-block significantly 👍🏼